README.md

```diff
@@ -46,23 +46,26 @@ Receptor is hosted on AWS at *(In progress)*
          ```
          brew services start postgresql
          ```
-
+     4. To get Python working with Postgres, install psycopg2
+          ```
+          pip install psycopg2
+          ```
     </details>
 
 5. Create the receptor database
     1. Open the psql shell
         ```
         psql postgres
         ```
-    2. Create 'receptor' databse
+    2. Create 'receptor' databse. 
         ```
         CREATE DATABASE receptor;
         exit;
         ```
 
 6. Make migrations, migrate and create a superuser
      ```
-     python manage.py makemigrations
```

Comment:
mmm I might be wrong here, but did you have to do `python manage.py makemigrations` as well to migrate user accounts? Or was `python manage.py makemigrations oligos` enough to run everything as well as migrate oligos app?

Reply:
Just `python manage.py makemigrations oligos` and that's the addition I've made. I don't see any repetition of that line. Or do you mean if we need `python manage.py migrate`?

Reply:
I meant if you need makemigrations only as well, without oligos, for migration the user model. When you run `python manage.py showmigrations` what do you see?